actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -433,7 +496,10 @@ private void validateMentors(CourseBatch courseBatch) {
     if (!dbParticipants.isEmpty()) {
       removeParticipants(dbParticipants, batchId, courseBatchObject.getCourseId());
     }
-    return finalParticipants;
+    participantsList.put(JsonKey.REMOVED_PARTICIPANTS, dbParticipants);
+    participantsList.put(JsonKey.UPDATED_PARTICIPANTS, updatedParticipants);
+    participantsList.put(JsonKey.PARTICIPANTS, finalParticipants);
```

Comment:
finalParticipants and updatedParticipants is the same. Then why insert two same lists?

Reply:
No sir, they are not same final participant have users which are being not removed + new users added.
but updated participants have users which are added newly only .